torch/testing/_internal/common_methods_invocations.py

```diff
@@ -108,7 +123,9 @@ def __init__(self,
                  promotes_integers_to_float=False,  # whether op promotes unary output to float or not
                  sample_inputs_func=None,  # function to generate sample inputs
                  aten_name=None,  # name of the corresponding aten:: operator
-                 supports_sparse=False  # supported for sparse
+                 aliases=None,  # list of aliases, e.g. torch.absolute -> torch.abs
+                 supports_sparse=False,  # whether the op correctly handles sparse input
```

Comment:
If suppost_sparse is inferable from the presence of a sparse_sample_inputs function then we should infer the property in the constructor instead of requiring the user pass it correctly.

I think it's inferable?